features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.infer.json

```diff
@@ -92,7 +92,7 @@
       "console.certificates.enabled": true,
       "console.secrets_management.enabled": true,
       "console.oidc_scopes.enabled": true,
-      "console.remote_fetch.enabled": true, 
+      "console.remote_fetch.enabled": false,
```

Comment:
Disable this feature from default.json file. We should not show it in the console UI

Reply:
Also we don't need this additional_features config in the infer.json introduced by previous PR

Reply:
Resolved with [4ac15cf](https://github.com/wso2/carbon-identity-framework/pull/4167/commits/4ac15cf3d190bc8dce0d7bee0377bde8a118a7ac)